core/gear_log/file_handle.ex

```diff
@@ -20,56 +20,57 @@ defmodule AntikytheraCore.GearLog.FileHandle do
     end
   end
 
-  @opaque t :: {Path.t, Time.t, File.io_device}
+  @opaque t :: {Path.t, Time.t, File.io_device, boolean}
 
-  defun open(file_path :: Path.t) :: t do
+  defun open(file_path :: Path.t, opts \\ []) :: t do
+    write_to_terminal = Keyword.get(opts, :write_to_terminal, determine_write_to_terminal())
```

Comment:
How about changing the variable name to `write_to_terminal?`?
In Elixir, it is common to add `?` to the suffix of a boolean variable name.

For example, in the following `if` statement, it is difficult to tell whether it is a variable name or a function that forgets the parentheses.
`if write_to_terminal, `

Reply:
Ok, I'll try.

Reply:
Changed. a70974e362fd2831f49cec496f94d8df1f5c256e